pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -485,6 +490,12 @@ private BrokerResponseNative handleSQLRequest(long requestId, String query, Json
     return brokerResponse;
   }
 
+  private String getBrokerTenant(String tableNameWithType) {
+    TableConfig tableConfig = _tableCache.getTableConfig(tableNameWithType);
+    Preconditions.checkNotNull(tableConfig, "Table config is not available for table '%s'", tableNameWithType);
```

Comment:
If you add this as Preconditions, it will throw exception in case the cache returns null. Better to add the string "Unknown_tenant" or something like that if the cache turns out to be null.

Reply:
done